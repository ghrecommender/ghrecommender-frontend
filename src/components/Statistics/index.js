// @flow strict

import React from 'react'
import { Statistic } from 'semantic-ui-react'

const items = [
  { label: 'Users', value: '19,925,204', key: 'users' },
  { label: 'Repositories', value: '64,245,061', key: 'repositories' },
  { label: 'Stars', value: '81,302,007', key: 'stars' },
]

const StatisticGroups = () => (
  <Statistic.Group items={items} size='small' inverted />
)

export default StatisticGroups
