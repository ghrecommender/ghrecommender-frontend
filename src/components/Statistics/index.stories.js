import React from 'react'
import 'semantic-ui-css/semantic.min.css'

import Statistics from './index'

const style = { backgroundColor: '#1b1c1d', width: '100%', height: '100%', position: 'absolute' }

export default {
  title: 'Components|Statistics',
  decorators: [story => <div style={style}>{story()}</div>],
}

export const statistics = () => <Statistics />
