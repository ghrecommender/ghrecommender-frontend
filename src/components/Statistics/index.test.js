import React from 'react'
import renderer from 'react-test-renderer'

import Statistics from './index'



test('<Statistics />', () => {
  const component = renderer.create(<Statistics />)
  expect(component.toJSON()).toMatchSnapshot()
})
