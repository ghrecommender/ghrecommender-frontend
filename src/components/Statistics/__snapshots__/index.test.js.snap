// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Statistics /> 1`] = `
<div
  className="ui small inverted statistics"
>
  <div
    className="ui statistic"
  >
    <div
      className="value"
    >
      19,925,204
    </div>
    <div
      className="label"
    >
      Users
    </div>
  </div>
  <div
    className="ui statistic"
  >
    <div
      className="value"
    >
      64,245,061
    </div>
    <div
      className="label"
    >
      Repositories
    </div>
  </div>
  <div
    className="ui statistic"
  >
    <div
      className="value"
    >
      81,302,007
    </div>
    <div
      className="label"
    >
      Stars
    </div>
  </div>
</div>
`;
